app/src/main/java/com/zulip/android/LoginActivity.java

```diff
@@ -198,6 +198,7 @@ public void onTaskFailure(String result) {
                     // Invalidate the token and try again, unless the user we
                     // are authenticating as is not registered or is disabled.
                     if (!loginTask.userDefinitelyInvalid) {
+                        GoogleAuthUtil.invalidateToken(that, token);
```

Comment:
update: i squashed this into the commit that removes this portion of the code entirely.